### ISSUE TITLE
Phantom iambic input

### ISSUE THREAD

**Issue by nealey** [nealey/vail#32]
Several users have commented on phantom iambic input, where they let go of the key but it keeps sending.

I've had two of them send a video and can confirm. This must be frustrating as all get-out.

I've never seen this, and it only seems to show up for them when using the vband adapter at a high WPM setting on the iambic keyer. Since the vband adapter only sends left and right control keyboard events, I'm guessing it's not the adapter, but rather that these folks are able to key very rapidly on their input devices, and this is only triggered by such rapid keying.

**Comment by nealey** [comment]
While investigating this, I was informed that my "typeahead" functionality in the Iambic keyer is not actually how things work. If you key faster than the iambic keyer, it doesn't queue up unsent entries. It just drops anything you aren't holding at the exact moment the pulse comes around.

I have a vague recollection that some previous user complained that it *wasn't* enqueueing things, which is why I implemented typeahead. So maybe we need a new UI control for this.

The reason I bring this up is that my current hypothesis is that the people seeing this problem have devices that are bouncing and queueing up lots of transmissions that the user doesn't feel they entered. I've disabled typeahead in a testing version of vail to see if that fixes things for these folks.
